hr_expense_invoice/models/hr_expense.py

```diff
@@ -27,56 +27,31 @@ class HrExpense(models.Model):
         comodel_name="account.move",
         inverse_name="source_invoice_expense_id",
     )
+    amount_residual = fields.Monetary(
```

Comment:
Why do you need this extra field? If there's a strong reason, put a comment above documenting the reason to add it.

Reply:
I already add the reason.